README.md

```diff
@@ -1,11 +1,15 @@
-# The PYthon Microscopy Environment
+# <img alt="PYME" src="PYME/resources/icons/pymeLogo.png" style="float:left"> The PYthon Microscopy Environment
 
-The PYthon Microscopy Environment provides image acquisition and data analysis functionality for a number of  widefield microscopy applications, but with a particular emphasis on single molecule localisation microscopy (PALM/STORM). The package is multi platform, running on Windows, Linux, and OSX.
-
-For more info see the project website at [http://www.python-microscopy.org].
+[![Azure](https://dev.azure.com/davidbaddeleynz/pyme-ci/_apis/build/status/azure-pipeline%davidbaddeleynz.pyme-ci)](
+    https://dev.azure.com/davidbaddeleynz/pyme-ci/_build/latest?definitionId=5)
 
-For installation instructions see [http://www.python-microscopy.org/doc/Installation/InstallationWithAnaconda.html]
+The PYthon Microscopy Environment provides image acquisition and data analysis functionality for a number of  widefield microscopy applications, but with a particular emphasis on single molecule localisation microscopy (PALM/STORM). The package is multi platform, running on Windows, Linux, and OSX.

```

Comment:
Every time I read this I feel like we start by putting ourselves in a box. What do we think of keeping the description short and leaving details of what we currently have to subsections of e.g. PYMEAcquire where we say we currently only support camera-based systems, etc.? When I got started with PYME the only people I knew who used it were deconvolving STED images. I think our one-sentence description could stand to be a little less constrained.

PYME (PYthon Microscopy Environment) is a collection of programs which together constitute a seamless pipeline of image acquisition, data storage, visualization, and analysis for (light?) microscopy applications. PYME components can be used individually if desired, and run on Windows, Mac, and Linux.

Reply:
How about shorter?

PYME (PYthon Microscopy Environment) is an open-source application suite for light microscopy image acquisition, data storage, visualization, and analysis.

Reply:
I like it